### PR TITLE
Added index and appendix to neuroscience

### PR DESCRIPTION
**Pull request by ckline-tryptic**
Issue: [5284](https://github.com/openstax/cnx-recipes/issues/5284)

### CODE REVIEW

**Comment by kswanson33** [comment]
looks like neuroscience spec files could use an update. you can update with
1. run neuroscience test locally with `rspec spec/recipes_spec/main_spec.rb:L#`
2. run `./scripts/update_recipes_spec_data`
3. check that `expected_output` for neuroscience has changed in the expected way

**Comment by ckline-tryptic** [comment]
Unfortunately, while fixing the expected output, ran into some errors with injected exercises update from main. Will keep an eye on main for the fix, then proceed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add `lifestage-context` note to `nursing-internal`
 * Add section titles links to `summary` in `nursing-internal`
 * Change EOC titles in `nursing-internal`
+* Added `index` and `appendix` to `neuroscience`
```
Comment (wilkus27) [inline CHANGELOG.md:31]: Sorry, first I approved but then I noticed problem with CHANGELOG. Please move this log to `Unreleased`. Versions that are already released shouldn't be changed.

**Review by wilkus27 — CHANGES_REQUESTED** [review]

**Review by kswanson33 — APPROVED** [review]

**Review by wilkus27 — APPROVED** [review]
Looks good :wink:

File: lib/recipes/neuroscience/recipe.rb [hunk 2]
```diff
@@ -65,13 +65,29 @@
...
+
   BakeChapterIntroductions.v1(book: book)
   BakeIframes.v1(book: book)
   BakeFootnotes.v1(book: book)
+  BakeIndex.v1(book: book)
   BakeCompositePages.v1(book: book)
   BakeCompositeChapters.v1(book: book)
   BakeToc.v1(book: book)
   BakeLinkPlaceholders.v1(book: book)
+  BakeIndex.v1(book: book)
```
Comment (wilkus27) [inline lib/recipes/neuroscience/recipe.rb:90]: It's added twice

**Pull request merged**